tests/unit/workflows/pipeline_store/test_upload.py

```diff
@@ -266,20 +225,10 @@ def test_upload_duplicate_record(
         assert potential_duplicates in caplog.text
 
 
-def test_force_upload_duplicate_record(zenodo_api_mocker: pytest_mock.MockerFixture):
-    zenodo_api = ZenodoAPI(sandbox=True)
-    workflow = ZenodoUploadWorkflow(
-        dpath_pipeline=TEST_PIPELINE,  # fMRIprep 24.1.1
-        zenodo_api=zenodo_api,
-        assume_yes=True,
-        force=True,
-    )
+def test_force_upload_duplicate_record(workflow: ZenodoUploadWorkflow):
+    workflow.assume_yes = (True,)
```

Comment:
This should be `True` without the tuple

```suggestion
    workflow.assume_yes = True
```